framework/src/transfers/MultiAppVariableValueSamplePostprocessorTransfer.C

```diff
@@ -41,15 +47,104 @@ MultiAppVariableValueSamplePostprocessorTransfer::validParams()
 MultiAppVariableValueSamplePostprocessorTransfer::MultiAppVariableValueSamplePostprocessorTransfer(
     const InputParameters & parameters)
   : MultiAppTransfer(parameters),
+    MeshChangedInterface(parameters),
     _postprocessor_name(getParam<PostprocessorName>("postprocessor")),
-    _from_var_name(getParam<VariableName>("source_variable")),
-    _comp(getParam<unsigned int>("source_variable_component"))
+    _var_name(getParam<VariableName>("source_variable")),
+    _comp(getParam<unsigned int>("source_variable_component")),
+    _var(_fe_problem.getVariable(0, _var_name))
 {
   if (_directions.size() != 1)
     paramError("direction", "This transfer is only unidirectional");
 
-  if (isParamValid("from_multi_app"))
-    paramError("from_multi_app", "This transfer direction has not been implemented");
+  if (_directions.contains("from_multiapp"))
+  {
+    // Check that the variable is a CONSTANT MONOMIAL.
+    auto & fe_type = _var.feType();
+    if (fe_type.order != CONSTANT || fe_type.family != MONOMIAL)
+      paramError("source_variable",
+                 "Variable must be in CONSTANT MONOMIAL when transferring from a postprocessor "
+                 "from sub-apps.");
+
+    if (!_fe_problem.getAuxiliarySystem().hasVariable(_var_name))
+      paramError("source_variable", "Variable must be an auxiliary variable");
+  }
+}
```

Comment:
i feel like this transfer should check the execution flags of the postprocessors. 
If you dont want to do it, let me know and i ll create an issue

Reply:
Not for this moment.

Reply:
created #20969